File: 프로그래머스/lv2/12941. 최솟값 만들기/최솟값 만들기.js
function solution(A,B){
    var answer = 0;
  A.sort((a,b) => a - b );
  B.sort((a,b) => b - a);
 // console.log(A,B)
  for(let i =0; i < A.length; i++){
    for(let j =0; j < B.length ; j++){
      if(i === j){
        answer += A[i] * B[j];
      }
    }
  }
  return answer;
}

File: 프로그래머스/lv2/12941. 최솟값 만들기/최솟값 만들기.py
def solution(A,B):
    answer = 0
    A.sort()
    B = sorted(B,reverse = True)
    for i in range(len(A)):
        # for j in range(len(B)):
        #     if i == j:
                answer += A[i] * B[i]
            
    return answer